presto-spi/src/main/java/com/facebook/presto/spi/type/RowType.java

```diff
@@ -149,17 +167,17 @@ public void writeObject(BlockBuilder blockBuilder, Object value)
         return fieldTypes;
     }
 
-    public List<RowField> getFields()
+    public List<Field> getFields()
     {
         return fields;
     }
 
-    public static class RowField
+    public static class Field
```

Comment:
You could extract this refactor (rename) change as separate commit, it would be easier to review this way.